gltfTutorial/gltfTutorial_005_MeshesTexturesImagesSamplers.md

```diff
@@ -0,0 +1,106 @@
+# Meshes, Textures, Images, Samplers
+
+
+## Meshes
+
+A [`mesh`](https://github.com/KhronosGroup/glTF/tree/master/specification#reference-mesh) describes a geometric object that appears in the scene. Every [`node`](https://github.com/KhronosGroup/glTF/tree/master/specification#reference-node) may refer to the meshes that are attached to the node, using their IDs:
+
+```javascript
+"nodes:" {
+    "exampleNode": {
+        "meshes" : [ "meshId0", "meshId1" ]
+        ...
+    }
+}
+```
+
+When the meshes are rendered, they are rendered with the [global transform](gltfTutorial_004_ScenesNodesCamerasAnimations.md#global-transforms-of-nodes) of the node that they are attached to. The same mesh may be "instanced" multiple times in a scene, by referencing it from different nodes that have different transforms.
+
+### Mesh primitives
+
+The mesh itself usually does not have any properties, but only contains an array of [`meshPrimitive`](https://github.com/KhronosGroup/glTF/tree/master/specification#reference-mesh.primitive) objects. These `meshPrimitive` objects are smaller parts or building blocks of a larger object:  

```

Comment:
The spec uses backticks throughout.  I would probably do the same here.  For plural, I usually do, for example, "`accessor` objects."  It's the best workaround I could come up with.

My main point was that `meshPrimitive` is not in glTF; it is `mesh.primitive`.